comps/imagegen/triton/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+build:
+	docker build -t base-installer-ubuntu22.04:1.15.1-15 -f Dockerfile.base .
+	docker build -t base -f Dockerfile.ubuntu .
+	docker build --progress=plain -t ohio-stability-triton -f Dockerfile .
```

Comment:
I'd add place holder parameters for `proxies` otherwise users behind proxy won't be able to run this out of box and need to modify the script.